Userland/Libraries/LibWeb/HTML/HTMLObjectElement.cpp

```diff
@@ -20,6 +20,9 @@ HTMLObjectElement::HTMLObjectElement(DOM::Document& document, DOM::QualifiedName
     : BrowsingContextContainer(document, move(qualified_name))
 {
     set_prototype(&Bindings::cached_web_prototype(realm(), "HTMLObjectElement"));
+
+    // - the element is created,
```

Comment:
This looks like it's from a spec. Can you add a link to this spec section in a comment?

Reply:
The spec doesn't have an anchor for this section. And this is a weird situation, since the spec lists triggers, but these triggeres are scattered throughout the implementation. I don't see a clean way to reference this spec section at each location.

The spec is [https://html.spec.whatwg.org/multipage/iframe-embed-object.html#the-object-element](https://html.spec.whatwg.org/multipage/iframe-embed-object.html#the-object-element) (the comment is from the section starting with `Whenever one of the following conditions occur:`).

Reply:
I see. That's a pain. Maybe do something like:

```c++
// https://html.spec.whatwg.org/multipage/iframe-embed-object.html#the-object-element
// Whenever one of the following conditions occur:
// - the element is created,
// ...the user agent must queue an element task on the DOM manipulation task source given the object element to run the following steps to (re)determine what the object element represents. This task being queued or actively running must delay the load event of the element's node document.
queue_element_task_to_run_object_representation_steps();
```

(But nicely wrapping that last bit.)

The idea is, someone who looks at this in the future should be able to locate the part of the spec for it, and see if the code matches or the spec has changed, etc.